streaming/streaming.go

```diff
@@ -100,11 +104,37 @@ func VerifyAndUpdateCfg(streamingCfg *StreamingCfg, dbName string) error {
 	if dfCfg.JobName == "" {
 		// Update names to have more info like dbname.
 		jobName, err := utils.GenerateName("hb-dataflow-" + dbName)
+		jobName = strings.Replace(jobName, "_", "-", -1)
 		if err != nil {
 			return fmt.Errorf("error generating stream name: %v", err)
 		}
 		streamingCfg.DataflowCfg.JobName = jobName
 	}
+
+	filePath := streamingCfg.TmpDir
+	if filePath[len(filePath)-1] != '/' {
+		filePath = filePath + "/"
+		streamingCfg.TmpDir = filePath
+	}
+	u, err := url.Parse(filePath)
+	if err != nil {
+		return fmt.Errorf("parseFilePath: unable to parse file path %s", filePath)
+	}
+	if u.Scheme != "gs" {
+		return fmt.Errorf("not a valid GCS path: %s, should start with 'gs'", filePath)
+	}
```

Comment:
Most of the code here seems duplicated with utils.go would it make sense to move it into a method?